cmd/terramate/cli/project.go

```diff
@@ -231,10 +231,6 @@ func (p *project) checkLocalDefaultIsUpdated() error {
 		Str("workingDir", p.wd).
 		Logger()
 
-	if !p.isDefaultBranch() {
```

Comment:
hmm so now the function is not exactly "checkLocalDefaultIsUpdated" no ? maybe "checkBranchIsUpdated" ?

Reply:
current code checks if HEAD and remote origin/main has the same ancestor commit, or in other words, it checks if remote origin/main commit is reachable from HEAD.
So the branch name is not important at all, but I agree we should improve the naming here.

But the current name `checkLocalDefaultIsUpdated()` is not technically wrong, because the code fetches the remote origin/main commit and verify if HEAD can reach it, if not, it's because the LocalDefaultBranch is not updated...